Revit_Core_Engine/Query/ColumnLine.cs

```diff
@@ -0,0 +1,73 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2022, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.Engine.Geometry;
+using BH.oM.Base.Attributes;
+using BH.oM.Geometry;
+using BH.oM.Physical.Elements;
+using System;
+using System.ComponentModel;
+
+namespace BH.Revit.Engine.Core
+{
+    public static partial class Query
+    {
+        /***************************************************/
+        /****              Public methods               ****/
+        /***************************************************/
+
+        [Description("Extracts the location line of a BHoM Column object.")]
+        [Input("column", "A BHoM Column object.")]
+        [Output("line", "The location line of the BHoM Column object.")]
```

Comment:
Not sure if that has been stressed enough in the onboarding sessions @vietle-bh, but we are trying to be specific in the descriptions, assuming that it is worth spending a bit more time on creation to avoid confusion in the future. As long as it is not always the case for the existing code base (unfortunately), I think it is a good custom when creating new stuff. I have made some suggestions below, but feel free to phrase it as you like - I am happy as long as the information is conveyed in an understandable form :wink:

```suggestion
        [Description("Extracts the location line of a BHoM Column object in preparation to push to Revit. Includes validity checks and flipping reversed nodes.")]
        [Input("column", "A BHoM Column object to extract the line from.")]
        [Output("line", "Preprocessed location line of the BHoM Column object to be used on push to Revit.")]
```

Reply:
Thanks. I'll make sure to do the same for future descriptions!